cli/modify_existing_test.go

```diff
@@ -301,9 +301,9 @@ yOTl4wVLQKA6mFvMV9o8B9yTBNg3mQS0vA==
 func removeVolatileEntries(input string) string {
 	var output strings.Builder
 	for _, line := range strings.Split(input, "\n") {
-		if strings.HasPrefix(line, "      checksum:") ||
-			strings.HasPrefix(line, "      modified:") ||
-			strings.HasPrefix(line, "\trootfs-image.checksum:") {
+		if strings.Contains(line, " checksum:") ||
+			strings.Contains(line, " modified:") ||
+			strings.Contains(line, "rootfs-image.checksum:") {
```

Comment:
This commit (fixing the tests) should be squashed with the first one (changing the output).